### ISSUE TITLE
Character count count message is not associated with textarea

### ISSUE THREAD

**Issue by 36degrees** [alphagov/govuk-frontend#1106]
The character count info message is (I believe) meant to be associated with the textarea using aria-describedby. However, this association is only made if there is no existing error message:

https://github.com/alphagov/govuk-frontend/blob/fe36aa381f79dc4ebd2f6cbea65381972179a696/src/components/character-count/character-count.js#L94-L105

We always provide an existing info message as of 7b534bf734cd0a1be2cff34d64018aa89abdabb6:

https://github.com/alphagov/govuk-frontend/blob/fe36aa381f79dc4ebd2f6cbea65381972179a696/src/components/character-count/template.njk#L26-L28

This means that the association will never be created.

From testing with Voiceover / Safari, the limit is still announced when it changes (because of the `aria-live` on the message), but it is no longer announced when focussing the field which I believe it should be.

**Comment by alex-ju** [comment]
Steven Proctor reported this [a while ago](https://github.com/alphagov/govuk-design-system-backlog/issues/67#issuecomment-440316495) and, at least at that point, an update on the textarea component was needed to be able pass the described by attribute to the element.

**Comment by 36degrees** [comment]
This was fixed by #1347
